esx_service/vmdk_ops.py

```diff
@@ -252,31 +253,57 @@ def getVMDKUuid(vmdk_path):
     except:
         return None
 
+# Create and return the devFS path for a VSAN object UUID.
+def get_vsan_devfs_path(uuid):
+
+    logging.debug("Got volume UUID %s", uuid)
+
+    # Objtool creates a link thats usable to 
```

Comment:
Pls add to the comment something like "please note that simply deleting the VMDK and backing object does not remove the link, so it needs to be taken care of explicitly after use"